godtools/ViewControllers/Tract/TractViewController.swift

```diff
@@ -8,13 +8,45 @@
 
 import Foundation
 import UIKit
+import SWXMLHash
 
 class TractViewController: BaseViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
         
         displayTitle()
+        initializeView()
+        setupSwipeGestures()
+    }
+    
+    func setupSwipeGestures() {
+        let swipeLeft = UISwipeGestureRecognizer(target: self, action: #selector(handleGesture))
+        swipeLeft.direction = .left
+        self.view.addGestureRecognizer(swipeLeft)
+        
+        let swipeRight = UISwipeGestureRecognizer(target: self, action: #selector(handleGesture))
+        swipeRight.direction = .right
+        self.view.addGestureRecognizer(swipeRight)
+    }
+    
+    @objc fileprivate func handleGesture(gesture: UISwipeGestureRecognizer) {
+        if gesture.direction == .right {
+            moveToPreviousPage()
+        } else if gesture.direction == .left {
+            moveToNextPage()
+        }
+    }
+    
+    func moveToNextPage() {
```

Comment:
I got confused, just in case, when moving to the next page is like moving from the 1st spiritual law to the 2nd, and from the 2nd to the 3rd, and each viewcontroller (TractViewController) is going  to have the collection of the views (for the tabs). Is that what you were talking about? or am I missing something?